dev/com.ibm.ws.org.apache.myfaces.2.3/bnd.bnd

```diff
@@ -146,35 +150,21 @@ Import-Package: \
   *
 
 -buildpath: \
-  org.apache.myfaces.core.impl;strategy=exact;version=2.3.3,\
+  org.apache.myfaces.core.impl;strategy=exact;version=${Implementation-Version},\
   org.apache.myfaces.buildtools:myfaces-builder-annotations;version=1.0.9,\
   com.ibm.ws.jsf.shared;version=latest,\
   com.ibm.websphere.javaee.jsf.2.3;version=latest,\
-  com.ibm.websphere.javaee.jstl.1.2;version=latest,\
-  com.ibm.websphere.javaee.persistence.2.1;version=latest,\
-  com.ibm.ws.org.apache.commons.digester.1.8;version=latest,\
-  com.ibm.ws.org.apache.commons.beanutils.1.8.3;version=latest,\
-  com.ibm.ws.com.google.guice.2.0;version=latest,\
   com.ibm.ws.classloading;version=latest,\
   com.ibm.ws.webcontainer;version=latest,\
   com.ibm.ws.serialization;version=latest,\
   com.ibm.ws.container.service;version=latest,\
   com.ibm.ws.adaptable.module;version=latest,\
-  com.ibm.ws.javaee.dd;version=latest,\
-  com.ibm.ws.javaee.dd.common;version=latest,\
   com.ibm.ws.anno;version=latest,\
   com.ibm.ws.managedobject;version=latest,\
   com.ibm.ws.jsp;version=latest,\
-  com.ibm.ws.org.apache.commons.collections;version=latest,\
-  com.ibm.ws.beanvalidation.core;version=latest,\
   com.ibm.websphere.javaee.el.3.0;version=latest,\
-  com.ibm.websphere.javaee.jsp.2.3;version=latest,\
   com.ibm.websphere.javaee.servlet.4.0;version=latest,\
   com.ibm.websphere.javaee.cdi.2.0;version=latest,\
-  com.ibm.websphere.javaee.ejb.3.2;version=latest,\
-  com.ibm.websphere.javaee.websocket.1.1;version=latest,\
-  com.ibm.websphere.javaee.validation.2.0;version=latest,\
-  com.ibm.ws.cdi.interfaces;version=latest,\
   com.ibm.ws.logging.core,\
```

Comment:
Just more of a comment here for myself, I assume we had these added at one point in time due to some overlays that we had and once we removed those overlays and the source did not need to be compiled any longer they just lingered. So I don't see an issue with removing these.

Reply:
That was my understanding.  I went through these one-by-one to see what was actually required at build time.